docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+database:
+  image: "sameersbn/mysql:latest"
+  environment:
+    DB_USER: staytus
+    DB_PASS: staytus
```

Comment:
The database container is only reachable on the host system, not publicly on the internet. So this should only be a problem, if the webserver is breached and someone can access the host network.

I don't think `docker-compose` offers a way to generate a random password. But we could add a starting script to replace the password with a secure one and then start it for them, how about that?